app/routes/wow.weekly-price-group-delta.tsx

```
@@ -0,0 +1,1256 @@
+            {getOribosLink(wowRealm.name, 'TUJ', wowRegion)({ row })}
+          </div>
+        )
+      }
+    }
+  ]
+
+  // Add performance threshold control
+  const renderPerformanceControl = () => (
+    <div className="mb-4 flex items-center gap-2">
+      <label className="text-sm font-medium">Hide items below</label>
+      <select
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)